public/content/about/index.md

```diff
@@ -119,6 +119,10 @@ Make sure you read [our principles](/contributing/design-principles/) and [our s
 
 We welcome feedback on our design principles, design system and the style guide. Remember, ethereum.org is for the community, by the community.
 
+## License {#license}
+
+The ethereum.org website is open source and built under an [MIT License](https://github.com/ethereum/ethereum-org-website/blob/dev/LICENSE) unless otherwise specified.
```

Comment:
Should we also link out to the [terms-of-use page](https://ethereum.org/en/terms-of-use/)? (From this page we would link to `/terms-of-use/`)

Reply:
```suggestion
The ethereum.org website is open source and built under an [MIT License](https://github.com/ethereum/ethereum-org-website/blob/dev/LICENSE) unless otherwise specified. More on [terms of use](/terms-of-use/) of ethereum.org.
```

Agree here, this would be my suggestion.